### DISCUSSION TITLE
[BUG] AppImage doesn't work on Linux Mint 21

### DISCUSSION THREAD

**Discussion by ppazos** [srevinsaju/guiscrcpy#349] [General]
**Describe the bug**
Executed both available AppImage files and I get no screen, in the terminal there is this error:

error while loading shared libraries: libavfilter.so.6: cannot open shared object file: No such file or directory

**To Reproduce**
1. Download AppImage
3. Give execution permissions
4. Execute
5. Start scrcpy
6. Just the menus appear, no screens.

Executed srccpy directly and I get a beautiful screen on my desktop.

**Expected behavior**
The screen should appear.

**Screenshots**

![Screenshot_2022-09-26_15-36-06](https://user-images.githubusercontent.com/1278556/192354310-2f14e2c2-3d0a-49aa-9c31-b9af10608298.png)

**Desktop (please complete the following information):**
* [ ] Windows
- **Architecture**: Example: 32-bit / 64-bit
- **Version**: Example: 10, 8.1, 7

* [x] Linux
 - Please paste your `neofetch` here. (What is a [neofetch](https://github.com/dylanaraps/neofetch))
A `neofetch` is always available in your package repositories. 

- > If you do not want to install neofetch; Please provide the name of your desktop environment, distro and architecture

* [ ] MacOS
 - **Version**: Example Catalina 10.15

**Comment by srevinsaju** [comment]
Looks like `libavfilter` is not installed on your host / the scrcpy bundled is not compatible with your OS. You will need to either manually install libavfilter, or install scrcpy on your host.

**Comment by ppazos** [comment]
Hi @srevinsaju which package contains libavfilter?

On the other hand, shouldn't this program come with all the dependencies? It will have a lot of  variability between linux distros if not all dependencies are included.

**Comment by srevinsaju** [comment]
> On the other hand, shouldn't this program come with all the dependencies? It will have a lot of variability between linux distros if not all dependencies are included.

I haven't packaged this for all distros. If you would like a solution which works out of the box, the [Flatpak](https://flathub.org/apps/details/in.srev.guiscrcpy) might be a good call.
